client/src/pages/Profiles/Profiles.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button, TextField, Typography } from '@mui/material';
+import useStyles from './useStyles';
+import hero from '../../images/landing/hero.jpg';
+import { useEffect, useState } from 'react';
+import { getProfiles } from '../../helpers/APICalls/getProfiles';
+
+export default function Profiles() {
+  const classes = useStyles();
+  const [profiles, setProfiles] = useState([]);
+
+  useEffect(() => {
+    getProfiles().then((resp) => setProfiles(resp.profiles));
+  }, []);
+
+  const [currentIndicesTopRow, setCurrentIndicesTopRow] = useState(3);
+  const [currentIndicesBottomRow, setCurrentIndicesBottomRow] = useState(6);
+
+  const Profile = ({ profile }: any) => {
+    return (
+      <>
+        <div className={classes.cardContainer}>
```

Comment:
Let's also make sure we use the Card API as each card contains information about a particular sitter

Ref: https://mui.com/api/card/